### PR TITLE
Refine Excel Graph error documentation

### PR DESCRIPTION
**Pull request by chiz-ms**
- Separated important second-level error codes with other (e.g., Rich API) second-level error codes;
- Refined the words to make instructions clearer;
- Added some missing details.

### CODE REVIEW

**Review by AmandaHan123 — APPROVED** [review]

File: concepts/workbook-error-handling.md [hunk 1]
```diff
@@ -63,71 +64,86 @@ The `innerError` object might recursively contain more `innerError` objects with
...
+All contents in the response that are not used in the above steps are for diagnostics purpose only (including strings in `message` fields). It is not recommended to make a dependency on these contents as they may change without notice.
 
 ## Special case handling
 
-For [sessionful requests](excel-manage-sessions.md#request-types), if you encounter a `502/badGateway` or `503/serviceUnavailable` error, when a second-level error code is listed in [Detailed error codes](workbook-error-codes.md#detailed-error-code), parse the second-level code and follow the corresponding instructions; otherwise, we reconmmend that you recreate the session directly.
+For [sessionful requests](excel-manage-sessions.md#request-types), if you encounter a `502/badGateway` or `503/serviceUnavailable` error, when a known second-level error code is found, follow the corresponding instructions; otherwise, we recommend that you recreate the session directly.
+
+## Example of other second-level error codes
+
+Here're some examples of other possible second-level error codes and corresponding handling instructions.
```
Comment (JarbasHorst) [inline concepts/workbook-error-handling.md:128]: The word "some" isn't specific and gives the impression that developers might find additional error codes that aren't documented. Why do you not provide a list of all examples? If this list contains all existing examples, then I recommend you to rephrase this sentence:

- The following list shows examples of uncommon second-level error codes that you might encounter along with the corresponding handling instructions for each error code.

**Comment by JarbasHorst** [comment]
Hello @chiz-ms, I reviewed your PR and left some comments. Please, take a look when you will have time.

**Also, to avoid merge conflicts, please merge PR [17163](https://github.com/microsoftgraph/microsoft-graph-docs/pull/17163) into your branch to include my fixes and before addressing my comments.**

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review. Please, merge PR [17287](https://github.com/microsoftgraph/microsoft-graph-docs/pull/17287) into this branch to include my final edits.

**Review by Lauragra — APPROVED** [review]

**Comment by JarbasHorst** [comment]
@Lauragra, please merge PR [17398](https://github.com/microsoftgraph/microsoft-graph-docs/pull/17398) into this branch to include a fix for an appearance of "there are".

File: concepts/workbook-error-handling.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ The following table lists instructions for important errors that Microsoft Graph
 
 >**Note:** For the regular pattern, the failed request is defined as the request corresponding to the response. For the long-running operation pattern, the failed request is the one that triggers the failed operation.
 
-For additional examples of other possible second-level error codes, see [Examples of other second-level error codes](#examples-of-other-second-level-error-codes).
+There are also other second-level error codes, which Microsoft Graph clients can optionally handle, or choose to fall back to [top-level error codes](#3-parse-the-top-level-error-code) or [status codes](#4-parse-the-status-code) instead. For examples of such second-level error codes and corresponding handling instructions, see [Examples of other second-level error codes](#examples-of-other-second-level-error-codes).
```
Comment (Lauragra) [inline concepts/workbook-error-handling.md:106]: Would it be more clear to revise as:
Microsoft Graph clients can also optionally handle second-level error codes, or fall back to [top-level error codes](#3-parse-the-top-level-error-code) or [status codes](#4-parse-the-status-code) instead. For examples of second-level error codes and instructions for how to handle them, see...

**Pull request merged**